templates/bake/Plugin/src/Plugin.php.twig

```diff
@@ -19,10 +19,43 @@ declare(strict_types=1);
 namespace {{ namespace }};
 
 use Cake\Core\BasePlugin;
+use Cake\Core\PluginApplicationInterface;
+use Cake\Routing\RouteBuilder;
 
 /**
  * Plugin for {{ namespace }}
  */
 class Plugin extends BasePlugin
 {
+    /**
+     * Load all the plugin configuration and bootstrap logic.
+     *
+     * The host application is provided as an argument. This allows you to load
+     * additional plugin dependencies, or attach events.
+     *
+     * @param \Cake\Core\PluginApplicationInterface $app The host application
+     * @return void
+     */
+    public function bootstrap(PluginApplicationInterface $app): void
+    {
+    }
+
+    /**
+     * Add routes for the plugin.
+     *
+     * @param \Cake\Routing\RouteBuilder $routes The route builder to update.
+     * @return void
+     */
+    public function routes(RouteBuilder $routes): void
+    {
+        $routes->plugin(
```

Comment:
Should we have a comment along the lines of 'if you have many routes you can create config/routes.php and delete this method'?

I personally like not having to sift through dozens of files to figure out how a plugin works but I can see folks also preferring smaller files that only do one thing.

Reply:
Adding a comment sounds fine.

Reply:
Comment added. This can be merged once the next CI run passes.